engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -51,6 +52,15 @@
     );
 
     /**
+     * Sometimes, a tracking system (i.e. for room-scale VR) provides a pose for the player hand position. In this
+     * case, special handling is necessary, and this accessor gives a way to check.
+     * @return true if this class is receiving tracking updates, false if not.
+     */
+    public boolean isTracked() {
+        return trackingDataReceived;
+    }
```

Comment:
I must admit that went over my head a bit. This is a part of the codebase I'm highly unfamiliar with. I'm just concerned with getting too much VR-related code peppering the codebase, to the point that it can't be easily extracted and then 1-2 years down the line somebody will have to do work similar to what you did to eradicate Oculus support, just to add support for some other VR platform.

Let's continue to keep this in mind and hopefully one day we'll get to a polish phase where we can look again at the big picture and find ways to smooth these issues out.